docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '3.8'
+services:
+  app:
+    build: .
+    container_name: amigos_ecom_app_c
+    image: app_amigos_ecom
+    ports:
+      - 3000:3000
+    environment:
+      - DB_NAME=amigos_ecom
+      - DB_USERNAME=postgres
+      - DB_USER_PASSWORD=test1234
+      - DB_HOST=db
```

Comment:
Can we pick those data from `.env` rather than hard-coding them in `docker-compose` file